docs/development/ROLE_README_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+theforeman.foreman.your_new_role
```

Comment:
I like that this strives for consistency.
But is it substantially different from what `a-galaxy role init` provides? Does it provide a README template?

Reply:
You can provide your own skeleton to reference (https://docs.ansible.com/ansible/latest/galaxy/dev_guide.html#using-a-custom-role-skeleton) which we could do to include this README and at least a tasks/main.yml. As most roles do use the common role variables and have a common structure with the examples etc. I do find myself copying other roles READMEs and editing (and always missing updating something when I do).